pkg/metrics/metrics.go

```diff
@@ -71,6 +71,9 @@ const (
 	// SubsystemAPILimiter is the subsystem to scope metrics related to the API limiter package.
 	SubsystemAPILimiter = "api_limiter"
 
+	// SubsystemNodeNeigh is the subsystem to scope metrics related to management of node neighbor.
```

Comment:
This commit is missing the `metrics.ArpingRequestsTotal.WithLabelValues(failed).Inc()` calls to actually populate the metrics, if we don't have them then there's no point in backporting the PR.